### DISCUSSION TITLE
Keycloak Roadmap for 2024

### DISCUSSION THREAD

**Discussion by MGLL** [keycloak/keycloak#26257] [Q&A]
Hello,

I wanted to known is there a Roadmap for 2024 planned for publication?

I'm interested to see what would be the next steps and expected evolution in the incoming year.

**Comment by Paul6552** [comment]
Welcome to keycloaks world, where everything is a guess :-D

There is already a thread: [Click me](https://github.com/keycloak/keycloak/discussions/17096)

**Reply by MGLL** [reply]
Yes I saw it :) but it was for "2023", so I'm I try to see if there will be one for "2024". 
Thanks

**Reply by tgau** [reply]
Any update for 2024/25?

**Comment by stianst** [comment]
All major or minor releases going forward are planned as [milestones](https://github.com/keycloak/keycloak/milestones), which has tentative dates on them.

By looking at open/closed issues for a given release you can get an idea on what's coming. However, I'm also planning to provide a more high-level roadmap at some point, which contains only highlights and not all the noise from issues.
